packages/threedimensional/src/components/threeDimensionalScene/ThreeDimensionalScene.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { ThreeDimensionalScene } from './ThreeDimensionalScene';
+
+export default {
```

Comment:
this story depends on uploading a file which makes it kind of _useless_, can we upload the file to a public place and reference it instead?

Reply:
Agree, I'll look into other ways of storing the asset so that it can be reused without uploading it everytime 👍🏼